README.md

```diff
@@ -11,8 +11,8 @@ in this project.
 
 ```
 plugins {
-    id 'eu.xenit.alfresco'
-    id 'eu.xenit.amp'
+    id 'eu.xenit.alfresco' version "0.1.3"
```

Comment:
```suggestion
    id 'eu.xenit.alfresco' version "0.1.3" // Have a look at https://plugins.gradle.org/plugin/eu.xenit.alfresco for the latest version
```

Reply:
I think @Jasper-Hilven should include this change first